### ISSUE TITLE
[BUG] NextJS previews do not work on Forest

### ISSUE THREAD

**Issue by cvanh** [forest-host/support#34]
**Describe the bug**
In NexJS there is a option to preview an page. When you try to preview an page on forest the IsPreview is false when it should be true. This error is only on forest localy everything works

**Include relevant information**
the isolated error is this: https://bitbucket.org/constantijnvanhaa/nextjs_cms/src/main/
the environment id: af385da2-1765-4445-a1a8-c7dd708f282e

**To Reproduce**
1. use my repo
2. use my blueprint https://pastebin.com/PWu47m2r
3. create the environment
4. set up wordpress
5. redeploy nodejs
6. go to wordpress 
7. create a draft post 
8. press preview

**Expected behavior**
nextjs router IsPreview being true

**Screenshots**

**Check for sensitive information**
If you find yourself wanting to share the name of a project, environment or domain, try to find the ID.
This prevents sharing sensitive information in a public space. You can always contact us on Slack or send a message to massim@forest.host.

**Comment by mass-im** [comment]
Thanks for reporting!
This sounds very specific. If you run the same setup locally (meaning running `node server.js`) do you end up with the same result?
I'm wondering if the way this is setup may cause the issue.

**Comment by cvanh** [comment]
yes locally everything works. The setup is from vercel so that should work otherwise other people would have reported it

**Comment by Forest-Gio** [comment]
could u send me the link of your environment in a private message so i can assist you further? 

If you need to add me to your team please invite giovanni@forest.host

**Comment by cvanh** [comment]
Gio and me couldnt figure out what the issue was. I am not shure what to do now, do you have any idea?

**Comment by cvanh** [comment]
I am running the server via node server.js.

if you could share the varnish config privately i can try to set up this issue in a docker container and share that with you.

**Comment by mass-im** [comment]
Install varnish and you're there ;)

**Comment by cvanh** [comment]
hi,

the previews still work when the next server is behind a caching layer, do you got any other idea what it could be?

<img width="1080" alt="Screenshot 2023-04-11 at 09 57 17" src="https://user-images.githubusercontent.com/39672378/231094565-8e51f6f8-ab7b-429e-8e3d-80e224e3d5fc.png">

**Comment by mass-im** [comment]
This should be fixed now!
